packages/design-picker/src/components/index.tsx

```diff
@@ -31,15 +31,25 @@ interface DesignPreviewImageProps {
 	design: Design;
 	locale: string;
 	highRes: boolean;
+	verticalId?: string;
 }
 
-const DesignPreviewImage: React.FC< DesignPreviewImageProps > = ( { design, locale, highRes } ) => {
+const DesignPreviewImage: React.FC< DesignPreviewImageProps > = ( {
+	design,
+	locale,
+	highRes,
+	verticalId,
+} ) => {
 	const scrollable = design.preview !== 'static';
 	const isMobile = useViewportMatch( 'small', '<' );
 
 	return (
 		<MShotsImage
-			url={ getDesignPreviewUrl( design, { language: locale, use_screenshot_overrides: true } ) }
+			url={ getDesignPreviewUrl( design, {
+				language: locale,
+				verticalId,
```

Comment:
[nit] Very minor nitpick since `DesignPreviewOptions` are mostly snake case, can we make `verticalId`, `vertical_id`? I'd like to change `siteTitle` to `site_title` eventually as well 😅

Reply:
Sure! I've changed them to `vertical_id` and `site_title`.